src/integration_tests/helpers/integration_helpers.py

```diff
@@ -181,30 +167,9 @@ def _create_remote_dataset(
         )
 
 
-def create_dataset_as_string(
-    filename: str, file_content: str, session: requests.Session, headers: dict[str, str]
-) -> None:
-    """
-    Method to create a remote dataset without parsing it as JSON.
-
-    Parameters:
-        filename: the filename to use for the file
-        file_content: the content of the file to be uploaded
-        session: a session instance for http/s connections
-        headers: the relevant headers for authentication for http/s calls
-
-    Returns:
-        None
-    """
```

Comment:
this function was weird, the way it was looking for a value that in some cases wasn't being passed or existed to determine the function had triggered. there are probably other ways to know it didnt trigger (like looking if it fails) we should maybe look into it

Reply:
Its probably written weirdly to work both local and gcp, but are moving away from local dev